src/Service/meeting_summary.py

```
@@ -24,13 +25,19 @@ def __init__(self):
+        try:
+            with open(self.file_path, 'r', encoding='utf-8') as f:
+                meeting_text = f.read()
         
-        formatted_prompt = self.prompt.format(회의내용=meeting_text)
-        output = self.model.invoke(formatted_prompt)
-        return output.content
+            formatted_prompt = self.prompt.format(회의내용=meeting_text)
+            output = self.model.invoke(formatted_prompt)
+            return output.content
+        except FileNotFoundError:
+            raise FileNotFoundError(f"Failed to read meeting notes from {self.file_path}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)